minecraft/index.html

```diff
@@ -251,19 +251,19 @@ <h2>东村</h2>
         <div class="container">
             <h2>相关文件</h2>
             <div class="container">
-                <a href="https://duckduckstudio.github.io/yazicbs.github.io/minecraft/excel/东海港规划.xlsx" target="_blank">[Excel]东海港规划</a>
+                <a href="data/东海港规划.xlsx" target="_blank">[Excel]东海港规划</a>
                 &nbsp;|&nbsp;
-                <a href="https://duckduckstudio.github.io/yazicbs.github.io/minecraft/excel/列车车站规划.xlsx" target="_blank">[Excel]列车车站规划</a>
+                <a href="data/列车车站规划.xlsx" target="_blank">[Excel]列车车站规划</a>
                 &nbsp;|&nbsp;
-                <a href="https://duckduckstudio.github.io/yazicbs.github.io/minecraft/excel/所需模组列表.xlsx" target="_blank">[Excel]世界所用模组列表</a>
+                <a href="data/mods.json" target="_blank">[JSON]世界所用模组列表</a>
```

Comment:
```suggestion
                <a href="data/mods.json" target="_blank" download>[JSON]世界所用模组列表</a>
```

这个 json 文件应该也是要要求下载。